app/ante/account.go

```diff
@@ -315,3 +291,40 @@ func (avd AccountAnteDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 
 	return next(ctx, tx, simulate)
 }
+
+func (avd AccountAnteDecorator) ethGasConsume(ctx *sdk.Context, acc exported.Account, accGetGas sdk.Gas, msgEthTx *evmtypes.MsgEthereumTx, simulate bool) error {
```

Comment:
why move ethGasConsume from func to method

Reply:
In `ethGasConsume`,  it need all of the there interface: `AccountKeeper`、`SupplyKeeper`、 `EVMKeeper` , there will be too much parameter if `ethGasConsume` is a function. So I change it to be a method, and it is logical in my opinion.